oauthdoctor/oauth/helper_test.go

```diff
@@ -0,0 +1,42 @@
+package oauth
+
+import (
+	"io/ioutil"
+	"log"
+	"testing"
+)
+
+func TestReadCustomerID(t *testing.T) {
+	log.SetOutput(ioutil.Discard)
```

Comment:
Is this to silence log output to stdout during testing?

Reply:
Yes